src/index.html

```diff
@@ -29,24 +30,26 @@
           <h2>octocat</h2>
 
           <ul>
-            <li>GitHub</li>
-            <li>San Francisco</li>
-            <li>email link</li>
-            <li>website</li>
-            <li>Joined on Jan 25, 2011</li>
+            <li id="company"><span class="octicon octicon-organization"></span>  GitHub</li>
+            <li id="location"><span class="octicon octicon-location"></span>  San Francisco</li>
+            <li id="email"><span class="octicon octicon-mail"></span>  email link</li>
+            <li id="blog"><span class="octicon octicon-link"></span>  website</li>
+            <li id="created_at"><span class="octicon octicon-clock"></span>  Joined on Jan 25, 2011</li>
```

Comment:
This gets the job done, for sure, but I bet you could refactor these into styles on your `<li>` elements instead of those ugly `<span>` elements. Those icons are just decorative, after all... I wonder how the Octicons work?